app/Scenes/Main.tscn

```diff
@@ -32,11 +26,33 @@ parameters/playback = SubResource( 1 )
 
 [node name="EnemySpawner" parent="." instance=ExtResource( 1 )]
 
-[connection signal="ChangedStamina" from="Player" to="HUD" method="_on_Player_ChangedStamina"]
-[connection signal="player_boid_count_update" from="Player" to="HUD" method="_on_Player_player_boid_count_update"]
+[node name="YSort" type="YSort" parent="."]
+
+[node name="RockTurtle" parent="YSort" instance=ExtResource( 9 )]
+position = Vector2( -282, 227 )
+
+[node name="RockTurtle2" parent="YSort" instance=ExtResource( 9 )]
+position = Vector2( -148, 381 )
+
+[node name="Player" parent="YSort" instance=ExtResource( 2 )]
+position = Vector2( -123, -127 )
+__meta__ = {
+"_edit_group_": true
+}
+
+[node name="AnimationPlayer" parent="YSort/Player" index="2"]
+root_node = NodePath("../../..")
+
+[node name="AnimationTree" parent="YSort/Player" index="3"]
+parameters/playback = SubResource( 1 )
+
+[node name="SpearLanceEnemy" parent="." instance=ExtResource( 10 )]
+position = Vector2( 210, -129 )
```

Comment:
can we use the enemy manager to spawn this instead?

Reply:
i think so